src/nupic/bindings/swig_proxy_preamble.py

```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------
```

Comment:
This is the boilerplate that gets prepended to _algorithms, _math, etc. python extension shared libraries.